src/main/java/org/cactoos/list/MapEntry.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.list;
+
+import java.util.Map;
+
+/**
+ * MapEntry as {@link java.util.AbastractMap.Entry}.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Fabricio Cabral (fabriciofx@gmail.com)
+ * @version $Id$
+ * @param <K> Key type
+ * @param <V> Value type
+ * @since 0.9
+ */
+public final class MapEntry<K, V> implements Map.Entry<K, V> {
+    /**
+     * The key.
+     */
+    private final K key;
+
+    /**
+     * The value.
+     */
+    private final V value;
+
+    /**
+     * Ctor.
+     * @param key The key of the entry's map.
+     * @param value The value associated to the key of the entry's map.
+     */
+    public MapEntry(final K key, final V value) {
+        this.key = key;
+        this.value = value;
+    }
+
+    @Override
+    public K getKey() {
+        return this.key;
+    }
+
+    @Override
+    public V getValue() {
+        return this.value;
+    }
+
+    @Override
+    // @checkstyle HiddenFieldCheck (1 line)
```

Comment:
@yegor256 I don't know why, qulice/checkstyle point an error because the method's parameter `value` has the same name of attribute `value`. If I'll change the code to:
```java
public V setValue(final V val)  {...}
```
the code works fine. So, I've two options:
1. To put the comment to qulice/checkstyle as above; or
2. To change the parameter's name.

I choosed option 1), because I'd like to get more semantic possible. WDYT?